File: ABC180/B_Various_distances/v1.scala
// AC

object Main extends App {
  val sc = new java.util.Scanner(System.in);
  val n = sc.nextInt();
  val xs = Array.fill(n)(sc.nextInt());
  val r1 = xs.map(x => (x max (-x)).toLong).sum;
  val r2 = Math.sqrt(xs.map(x => x.toLong * x).sum);
  val r3 = xs.map(x => (x max (-x)).toLong).max;
  println(r1);
  println(r2);
  println(r3);
}

File: abc171/e/v1.java
// AC

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    var sc = new Scanner(System.in);
    int n = sc.nextInt();
    var as = new int[n];
    for (int i = 0; i < n; i++) as[i] = sc.nextInt();
    int total = 0;
    for (int i = 0; i < n; i++) {
      total ^= as[i];
    }
    for (int i = 0; i < n; i++) {
      if (i > 0) System.out.print(" ");
      System.out.print(as[i] ^ total);
    }
    System.out.println();
  }
}
